src/main/java/net/minecraftforge/client/model/obj/ObjModel.java

```diff
@@ -687,6 +708,6 @@ public void bake(CompositeRenderable.PartBuilder<?> builder, IGeometryBakingCont
 
     public record ModelSettings(@NotNull ResourceLocation modelLocation,
                                 boolean automaticCulling, boolean shadeQuads, boolean flipV,
-                                boolean emissiveAmbient, @Nullable String mtlOverride)
+                                boolean emissiveAmbient, @Nullable String mtlOverride, boolean allowHomonymObjectName)
     { }
```

Comment:
This is a breaking change because any callers to the previous constructor (without the new parameter) will break. 

Please add a new constructor to this record which calls the canonical constructor with an appropriate default value for the new parameter, mark that new constructor as deprecated for removal (`@Deprecated(forRemoval = true, since = "1.19.2")`, and add a javadoc with a `@deprecated` tag directing users to the canonical constructor (which has the new parameter).